src/Microsoft.AspNetCore.Server.Kestrel.Https/HttpsConnectionAdapter.cs

```diff
@@ -118,13 +118,7 @@ private static X509Certificate2 ConvertToX509Certificate2(X509Certificate certif
                 return certificate2;
             }
 
-#if NETSTANDARD1_3
-            // conversion X509Certificate to X509Certificate2 not supported
-            // https://github.com/dotnet/corefx/issues/4510
```

Comment:
In core, the as cast *should* [always succeed](https://github.com/dotnet/corefx/blob/b1af73da70b59ae8e6967e50fbad6b2936bda009/src/System.Net.Security/src/System/Net/Security/SslStream.cs#L80-L94). I guess we could have removed this the last time you asked.  :man_shrugging: